mkdocs.yml

```
@@ -234,11 +234,12 @@ nav:
   - Contributing:
       - "dev_guides/index.md"
       - Contributing to Splink:
-        - Contributor Guide: "CONTRIBUTING.md"
-        - Building a Virtual Environment: "dev_guides/changing_splink/building_env_locally.md"
+        - Contributor Guide: "dev_guides/CONTRIBUTING.md"
+        - Development Quickstart: "dev_guides/changing_splink/development_quickstart.md"
         - Linting and Formatting: "dev_guides/changing_splink/lint_and_format.md"
         - Testing: "dev_guides/changing_splink/testing.md"
         - Building Docs: "dev_guides/changing_splink/build_docs_locally.md"
+        - Managing Dependencies: "dev_guides/changing_splink/managing_dependencies.md"
```

From - https://github.com/moj-analytical-services/splink/pull/2083

```suggestion
        - Managing Dependencies with Poetry: "dev_guides/changing_splink/managing_dependencies.md"
```